src/components/banner/Banner.js

```diff
@@ -48,10 +48,13 @@ Banner.propTypes = {
   onClose: PropTypes.func,
   /** If true, component will take the full width of it's container. */
   fullWidth: PropTypes.bool,
+  /** Size prop to add a padding to the Section or Island. */
+  size: PropTypes.string,
```

Comment:
You don't need to add it as a prop because it uses a `Section`/`Island` internally, which already supports the size.
Just adding it to the story knobs should be enough.